pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools>=57"]
```

Comment:
similarly to my comment on `package_requirements`, is this replacing `requirements.txt`? Do we need to move over the dependencies originally in that file?

Reply:
No, please refer to PEP 508 [here](https://peps.python.org/pep-0518/#file-format) for a more indepth breakdown